front/app/containers/Admin/pagesAndMenu/containers/Attachments/index.tsx

```diff
@@ -114,7 +114,7 @@ const AttachmentsForm = ({ intl: { formatMessage } }: InjectedIntlProps) => {
               },
               {
                 label: localize(customPage.attributes.title_multiloc),
-                linkTo: `${PAGES_MENU_CUSTOM_PATH}/${customPageId}/content`,
+                linkTo: adminCustomPageContentPath(customPageId),
```

Comment:
These make it difficult to tell what the actual path is. With a constant you can at least hover over it and see the string it evaluates to, for this you need to go to another file to actually figure out what you're linking to. I understand that there are some advantages but I'm not sure this is the right solution

Reply:
With the constant that was there you could see `string`, so you'd have to go to the file as well.

This function returns the same string. Advantage indeed is reducing chance of introducing bugs from multiple to 1 spot. Tbh, I don't get why we hardcode urls/path everywhere. Very error-prone.

But open to hear better solutions, so solutions other than previous hard-coded string that is.

Reply:
True, you can only see the path when the constant comes from the same file. I think an argument for hardcoded paths is that you can tell exactly what the URL is within the context of what you're editing. I certainly appreciate that when developing. At least for this branch, I don't see a danger of hardcoding paths since we're testing most of the navigation (by clicking buttons, not visiting URLs) in the [e2e tests](https://github.com/CitizenLabDotCo/citizenlab/blob/699d6a0502f7513ff80e81aa771cbe9e7552bb49/front/cypress/e2e/admin/pages_and_menu/create_update_and_view_custom_page.cy.ts). If we'd mistyped a path it would be pretty obvious when running these tests.

Maybe a discussion for the FE CDM. Would be interested to know if any bugs have been introduced due to mistyped paths